docs/source/developer/distributions.rst

```diff
@@ -0,0 +1,232 @@
+The :class:`~metasyn.distribution.ScipyDistribution` is a specialized base class for distributions that are based on
+`SciPy <https://docs.scipy.org/doc/scipy/index.html>`_ statistical distributions. 
+
+All the :mod:`~metasyn.distribution.datetime`, :mod:`~metasyn.distribution.discrete`, and :mod:`~metasyn.distribution.continuous` distributions are derived from this class.
+
+
+UniqueDistributionMixin class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The :class:`~metasyn.distribution.UniqueDistributionMixin` is a mixin class that can be combined with other distribution classes to create distributions that generate unique values.
+
+For example, the unique variants of the :class:`metasyn.distribution.regex.RegexDistribution` and the :class:`metasyn.distribution.faker.UniqueFakerDistribution` are implemented using this mixin as follows:
+
+.. code-block:: python
+
+    @metadist(implements="core.unique_regex", var_type="string", is_unique=True)
+    class UniqueRegexDistribution(UniqueDistributionMixin, RegexDistribution):
+
+.. code-block:: python
+
+    @metadist(implements="core.unique_faker", var_type="string")
+    class UniqueFakerDistribution(UniqueDistributionMixin, FakerDistribution):
+
+
+Metadist decorator method
+^^^^^^^^^^^^^^^^^^^^^^^^^
+When implementing a new distribution, the ``metadist`` decorator helps set the attributes of that distribution (e.g. ``implements``, ``var_type``, etc.). 
+
+**Parameters:**
+
+- ``implements``: A unique name for the distribution type, e.g. ``core.uniform`` or ``core.poisson``. 
+- ``var_type``: The type of variable associated with the distribution, e.g. ``discrete`` or ``continuous``.
+- ``provenance``: Information about the source (core, plugin, etc.) of the distribution.
+- ``privacy``: The privacy class or implementation associated with the distribution.
+- ``is_unique``: A boolean indicating whether the values in the distribution are unique.
+- ``version``: The version of the distribution. 
+
+.. admonition:: Note
+
+    Note that the parameters are the same as the attributes of the :class:`~metasyn.distribution.BaseDistribution` class.
+
+To use the ``metadist`` decorator, annotate the custom distribution class with ``@metadist``, passing in the attributes of the target distribution as parameters.
+
+For example, the following distributions use the decorator as follows:
+
+.. code-block:: python
+
+    @metadist(implements="core.multinoulli", var_type=["categorical", "discrete", "string"])
+    class MultinoulliDistribution(BaseDistribution):
+
+.. code-block:: python
+
+    @metadist(implements="core.unique_regex", var_type="string", is_unique=True)
+    class UniqueRegexDistribution(UniqueDistributionMixin, RegexDistribution):
+
+.. code-block:: python
+      
+    @metadist(implements="core.uniform_date", var_type="date")
+    class UniformDateDistribution(BaseUniformDistribution):
+
+
+The ``metadist`` decorator is implemented automatically as part of the main ``metasyn`` package. 
```

Comment:
What do you mean with automatically?

Reply:
I struggled with phrasing, but I mean that it's directly accessible as metasyn directly imports it from the subpackage. I have changed it to 

```
The ``metadist`` decorator, which is a part of the :mod:`metasyn.distribution.base` submodule, is directly accessible when importing the main metasyn package, as it's explicitly and relatively imported upon importing the main metasyn package.
```

Which I think is the proper terminology for this.